src/components/WhyWeDoIt/WhyWeDoIt.jsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+import heart from "./like-heart.svg";
+
+function WhyWeDoIt() {
+  return (
+    <div className="flex flex-col items-center w-52 h-64 shadow-md p-4 rounded-lg border">
```

Comment:
width should be 100%

Reply:
What do you mean 100%? Shouldn't be like figma?

Reply:
it should be one under one, I left comments for changes to be done to Figma, but no one did the changes so let us do it here

Reply:
![Screenshot (422)](https://user-images.githubusercontent.com/62112778/128094267-8cb7a901-3d12-46e9-a74d-fb8e1a38b756.png)
You mean like that right but a bit larger or a card with a full width ?